src/App.js

```diff
@@ -1,24 +1,11 @@
-import logo from './logo.svg';
 import './App.css';
+import { Todo } from './pages';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <>
```

Comment:
[Optional]
- [ ] Kindly use `React.fragment` tag instead of those empty ones to make your codes look more understandable :smiley: